src/tools/clippy/tests/ui/same_name_method.stderr

```diff
@@ -61,5 +61,17 @@ note: existing `foo` defined here
 LL |         impl T1 for S {}
    |         ^^^^^^^^^^^^^^^^
 
-error: aborting due to 5 previous errors
+error: method's name is the same as an existing method in a trait
+  --> $DIR/same_name_method.rs:75:13
+   |
+LL |             fn foo() {}
+   |             ^^^^^^^^^^^
+   |
+note: existing `foo` defined here
+  --> $DIR/same_name_method.rs:81:9
+   |
+LL |         impl T2 for S {}
+   |         ^^^^^^^^^^^^^^^^
```

Comment:
You need to modify the `.rs` file as well, don't you? Can you locally run `python ./x.py test src/tools/clippy` to confirm this is actually passing now?

Reply:
Yeah, I ran the clippy tests before creating the PR, it works without changing the `.rs` file. If I add the second `ERROR` line it also works. And if I add a third `ERROR` line then the test fails.

So I have updated with the second `ERROR` added, because that's what one would expect. I don't know why it was working without the second ERROR.

Reply:
We don't enforce `ERROR` annotations yet. Thanks for adding it!